doc/capi.txt

```diff
@@ -49,8 +49,14 @@ This is the easiest way of extending lxml at the C level.  A Cython_
 
     # My Cython extension
 
+    # directive pointing compiler to lxml header files;
+    # use ``aliases={"LXML_PACKAGE_DIR": LXML_PACKAGE_DIR}``
```

Comment:
Writing `LXML_PACKAGE_DIR` on both sides here seems more confusing than necessary. It's not very clear which refers to the `include_dirs` value below, and which the user needs to provide (and how). It should work to use `lxml.includes.__path__` for the alias value (assuming `lxml` is already installed at build time, which `setuptools` and `pip` can take care of).

Reply:
addressed. On my machine, `lxml.__path__` works, while `lxml.includes.__path__` does not, so I updated as requested but kept it pointing to lxml root directory

Reply:
Ah, right. The `api` C header files end up in that directory, so `lxml.__path__` is correct.